parlai/agents/image_seq2seq/modules.py

```diff
@@ -16,7 +16,7 @@
 from parlai.agents.transformer.modules import (
     TransformerGeneratorModel,
     TransformerEncoder,
-    _normalize,
+    normalize,
```

Comment:
note that i'm almost positive there are internal imports of this as well so we'd need to catch those

Reply:
True. I'll just have to have an internal PR ready to go before this lands. Though this function is going away anyway: https://github.com/facebookresearch/ParlAI/pull/3469

Reply:
uh oh